internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/FastClientDaVinciClientCompatTest.java

```diff
@@ -116,6 +119,8 @@ private DaVinciClient<String, TestValueSchema> setupDaVinciClient(String storeNa
             .put(CLIENT_USE_SYSTEM_STORE_REPOSITORY, true)
             .put(CLIENT_USE_DA_VINCI_BASED_SYSTEM_STORE_REPOSITORY, false)
             .put(DATA_BASE_PATH, dataPath)
+            .put(DAVINCI_P2P_BLOB_TRANSFER_PORT, TestUtils.getFreePort())
```

Comment:
I see we are adding these configs to many test files. Do we really use them in these files? Can we only add them if needed?

Reply:
Yes. The tests were breaking w/o them